src/ui/units/collections/components/CollectionActions/CollectionActions.tsx

```diff
@@ -135,29 +135,16 @@ export const CollectionActions = React.memo<Props>(
 
         return (
             <div className={b(null, className)}>
-                {collectionData && (
-                    <React.Fragment>
-                        {collectionsAccessEnabled && collectionData.permissions.listAccessBindings && (
-                            <Button className={b('access')} onClick={onEditAccessClick}>
-                                {i18n('action_access')}
```

Comment:
`action_access` and `action_edit` from i18n are no longer used, let's remove them

Reply:
upd: no need to remove them, they will be used in tooltips

Reply:
added tooltips